src/antsibull/cli/antsibull_lint.py

```diff
@@ -8,150 +8,20 @@
-                                         ' changelog config. Do not use this for Ansible'
-                                         ' collections!')
-
-        collection_docs = subparsers.add_parser('collection-docs',
-                                                parents=[common],
-                                                help='Collection extra docs linter for inclusion'
-                                                     ' in docsite. WARNING: this is deprecated;'
-                                                     ' use `antsibull-docs lint-collection-docs`'
-                                                     ' instead!')
-        collection_docs.set_defaults(func=command_lint_collection_docs)
-
-        collection_docs.add_argument('collection_root_path',
-                                     metavar='/path/to/collection',
-                                     help='path to collection (directory that includes galaxy.yml)')
-
-        if HAS_ARGCOMPLETE:
-            argcomplete.autocomplete(parser)
-
-        arguments = parser.parse_args(args[1:])
-
-        if getattr(arguments, 'func', None) is None:
-            parser.print_help()
-            return 2
-
-        normalize_toplevel_options(arguments)
-
-        verbosity = arguments.verbose
-        setup_logger(verbosity)
-
-        return arguments.func(arguments)
-    except SystemExit as e:
-        return e.code
-    except Exception:  # pylint: disable=broad-except
-        if verbosity > 0:
-            traceback.print_exc()
-        else:
-            print('ERROR: Uncaught exception. Run with -v to see traceback.')
-        return 1
-
-
-def command_lint_changelog(args: Any) -> int:
-    """
-    Validate a changelogs/changelog.yaml file.
-
-    :arg args: Parsed arguments
-    """
-    print(
-        'WARNING: `antsibull-lint changelog-yaml` is deprecated.'
-        ' Use `antsibull-changelog lint-changelog-yaml` instead.',
-        file=sys.stderr)
-
-    errors = lint_changelog_yaml(
-        args.changelog_yaml_path, no_semantic_versioning=args.no_semantic_versioning)
-
-    messages = sorted(set(f'{error[0]}:{error[1]}:{error[2]}: {error[3]}' for error in errors))
-
-    for message in messages:
-        print(message)
-
-    return 3 if messages else 0
-
-
-def command_lint_collection_docs(args: Any) -> int:
-    """
-    Validate docs/docsite/rst/ in a collection.
-
-    :arg args: Parsed arguments
-    """
-    print(
-        'WARNING: `antsibull-lint collection-docs` is deprecated.'
-        ' Use `antsibull-docs lint-collection-docs` instead.',
-        file=sys.stderr)
-
-    errors = lint_collection_extra_docs_files(args.collection_root_path)
-    errors.extend(lint_collection_links(args.collection_root_path))
-
-    messages = sorted(set(f'{error[0]}:{error[1]}:{error[2]}: {error[3]}' for error in errors))
-
-    for message in messages:
-        print(message)
-
-    return 3 if messages else 0
+    return 1
```

Comment:
Should this return `2` instead of `1`? According to `main()`'s docstring:

```
    Return codes:
        :0: Success
        :1: Unhandled error.  See the Traceback for more information.
        :2: There was a problem with the command line arguments
        :3: Linting failed
        :4: Needs to be run on a newer version of Python
```

Reply:
Good point. Changed in 1193368a24642d36ae8a484d980bb24ae9fa4f22.